app/jobs/fetch_news_job.rb

```diff
@@ -0,0 +1,36 @@
+class FetchNewsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(*args)
+    # Do something later
+    NewsFilter.all.each do |search_term|
+      puts search_term.search_term
+
+      @news = GoogleCustomSearchApi.search(search_term.search_term, page: 1)
```

Comment:
This should be wrapped with `begin` / `rescue` as it's making a call to an external API. The rescue should include sending a message to admins